gsl/multi_span

```diff
@@ -1488,7 +1488,7 @@ public:
     constexpr reference operator()(FirstIndex index, OtherIndices... indices)
     {
         index_type idx = {narrow_cast<std::ptrdiff_t>(index),
-                          narrow_cast<std::ptrdiff_t>(indices...)};
+                          narrow_cast<std::ptrdiff_t>(indices)...};
```

Comment:
@viboes Yes, you're a little ahead of me. I was just lazily using the PR to force the CI systems to go build with all the different compilers for me, in case I'd created some silly syntax error. ;-)